backend/testfiles/httphandler/injected-icon-post-roundtrip.test

```diff
@@ -1,5 +1,5 @@
 [http-handler POST /]
-PACKAGE.Darklang.Stdlib.Http.response request.body 200L
+PACKAGE.Darklang.Stdlib.Http.response (request.body) 200L
```

Comment:
TODO: in the grammar consider moving `field_access` to `simple_expression` so we don't have to wrap this in parens